tests/test_decision_maker/test_default.py

```diff
@@ -88,12 +88,11 @@ def role_from_first_message(  # pylint: disable=unused-argument
 class BaseTestDecisionMaker:
     """Test the decision maker."""
 
+    decision_maker_handler_cls = DecisionMakerHandler
+    decision_maker_cls = DecisionMaker
+
     @classmethod
-    def setup(
-        cls,
-        decision_maker_handler_cls=DecisionMakerHandler,
-        decision_maker_cls=DecisionMaker,
-    ):
+    def setup(cls):
```

Comment:
Is this change due to something in 3.10?

Reply:
@5A11 probably newer pytest causes this issue. old pytest does not work with python 3.10
so had to update this tests case